bears/stylus/StylintBear.py

```diff
@@ -36,12 +38,182 @@ class StylintBear:
     SEE_MORE = 'https://github.com/SimenB/stylint'
 
     @staticmethod
+    def generate_config(filename, file,
+                        allow_block_keyword: bool=False,
+                        allow_brackets: str='never',
+                        allow_colons: str='always',
+                        allow_colors: str='always',
+                        allow_spaces_after_commas: str='always',
+                        allow_spaces_after_comments: str='always',
+                        allow_trailing_whitespace: str='never',
+                        allow_css_literal: str='never',
+                        max_selector_depth: bool=False,
+                        check_duplicates: bool=True,
+                        allow_efficient_properties: str='always',
+                        set_extend_preference: bool=False,
+                        indent_size: bool=False,
+                        allow_leading_zero: str='never',
+                        set_max_errors: bool=False,
+                        set_max_warnings: bool=False,
+                        mixed_spaces_and_tabs: bool=False,
+                        set_naming_convention: bool=False,
+                        set_strict_naming_convention: bool=False,
+                        allow_none_keyword: str='never',
+                        check_no_important_keyword: bool=True,
+                        allow_spaces_inside_parens: bool=False,
+                        allow_placeholder: str='always',
+                        prefix_vars_with_dollar: str='always',
+                        allow_semicolons: str='never',
+                        sort_order: str='alphabetical',
+                        allow_stacked_properties: str='never',
```

Comment:
could you make all those `always` and `never` settings to bools? It seems that there are no more values^^